mem/memtable.go

```diff
@@ -93,12 +89,10 @@ func (m *Memtable) Write(batch writeBatch) (bool, error) {
 	var err error
 	batch.Range(func(key []byte, value []byte) bool {
 		if err = writeIter.Add(key, value, 0); err != nil {
-			if //goland:noinspection GoDirectComparisonOfErrors
-			err == arenaskl2.ErrRecordExists {
+			if errors.Is(arenaskl2.ErrRecordExists, err) {
```

Comment:
We deliberately use exact comparison here as this is performance sensitive code

Reply:
ok, got it, changed it back to original code

Reply:
maybe this can be marked in the comment alongside GoDirectComparisonOfErrors ?